app/src/main/res/values-ja/strings.xml

```diff
@@ -267,100 +267,5 @@
     <string name="all_day">常に通知する</string>
     <string name="next_day">(翌日)</string>
     <string name="schedule_disabled">全時刻</string>
-
-    <!-- Notification summary patterns  - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
-    <string name="new_messages_summary_pattern">\\d+ new messages</string>
-    <string name="new_messages_multiple_chats_summary_pattern">\\d+ (new )?messages from \\d+ chats</string>
-    <string name="notification_message_photo">(sent you a photo|📷 Photo)</string>
```

Comment:
You can actually include the pattern text in the translation. This is used to do pattern matching using a regex expression to identify what type of notification it is. These translations were provided for other languages by another helpful contributor. If you can figure out the translations for this that is ideal.